.github/workflows/sng-ingestor-shared-workflow.yml

```diff
@@ -89,6 +89,16 @@ jobs:
         with:
           service-account-key: ${{ secrets.gcloud-auth-staging }}
 
+      - name: Notify Slack if staging fails
+        uses: extenda/actions/slack-notify@v0
+        if: failure()
+        with:
+          text: |
+            *Validate failed for ${{ github.repository }}: ${{ github.workflow }}* :heavy_exclamation_mark:
+            Validate failed on ${{ github.event_name }} event. Workflow: ${{ github.workflow }}. Job: ${{ github.job }}. Run id: <https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}|${{ github.run_id }}>
+          channel: selfscan-pipeline-notifications
```

Comment:
Please make `channel` to be a input parameter with this channel as the default value.

Reply:
Done